File: first_page_module/src/main/java/com/mobile/first_page_module/model/IFirstPageModel.java
package com.mobile.first_page_module.model;

import java.util.Map;

import io.reactivex.Observable;
import okhttp3.ResponseBody;

/**
 * author    : xxxxxxxxxxx
 * e-mail    : xxxxxxxxxxx@qq.com
 * date      : 2019/3/10 16:57
 * introduce :
 */

public interface IFirstPageModel {

//    Observable<ResponseBody> firstPage(Map<String, String> bodyParams);

    Observable<ResponseBody> firstPageDetails(Map<String, String> bodyParams);


}

File: module_home/src/main/java/com/phone/module_home/model/IFirstPageModel.kt
package com.phone.module_home.model

import io.reactivex.Observable
import okhttp3.ResponseBody

/**
 * author    : Urasaki
 * e-mail    : 1164688204@qq.com
 * date      : 2019/3/10 16:57
 * introduce :
 */

interface IFirstPageModel {

    fun firstPage(bodyParams: Map<String, String>): Observable<ResponseBody>

    fun firstPageDetails(bodyParams: Map<String, String>): Observable<ResponseBody>

}